WcaOnRails/app/controllers/relations_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'relations'
```

Comment:
It worked fine without this locally, but appeared to fail on the staging. Any idea why isn't it loaded properly?